sdk/identity/azure-identity/src/test/java/LiveManagedIdentityTests.java

```diff
@@ -143,13 +152,18 @@ public void testManagedIdentityVmDeployment() {
 
     @Test
     @EnabledIfEnvironmentVariable(named = "AZURE_TEST_MODE", matches = "LIVE")
+    @Retry(maxRetries = 3)
     public void callGraphWithClientSecret() {
         Configuration configuration = Configuration.getGlobalConfiguration().clone();
 
-        String multiTenantId = configuration.get("AZURE_IDENTITY_MULTI_TENANT_TENANT_ID");
-        String multiClientId = configuration.get("AZURE_IDENTITY_MULTI_TENANT_CLIENT_ID");
+        String multiTenantId = "54826b22-38d6-4fb2-bad9-b7b93a3e9c5a";
+        String multiClientId = "4fc2b07b-9d91-4a4a-86e0-96d5b9145075";
         String multiClientSecret = configuration.get("AZURE_IDENTITY_MULTI_TENANT_CLIENT_SECRET");
 
+        System.out.println("Multi Tenant ID:" + multiTenantId);
+        System.out.println("Multi Client  ID:" + multiClientId);
+        System.out.println("Multi Client Secret:" + multiClientSecret);
```

Comment:
Probably should not log the secret?

Reply:
They are not logged as Devops redacts them,